app/src/ui/lib/filter-list.tsx

```diff
@@ -283,10 +288,6 @@ export class FilterList<T extends IFilterListItem> extends React.Component<
       return
     }
 
-    if (this.props.onFilterKeyDown) {
```

Comment:
I'm not sure I understand why this is being 🔥'd?

Reply:
I'm not sure either. Will restore it.

Reply:
Oh, now I know.

`onFilterKeyDown` is related to the filter state changing, giving users a chance to cancel this event (for example, <kbd>Esc</kbd> to close the foldout. Because `this.state.filter` isn't supported any more I need to test all the places where we use this and confirm we are correctly handling this still.